src/main/java/org/semux/core/Transaction.java

```diff
@@ -45,9 +45,13 @@
 
     private Signature signature;
 
+    private final long gas;
+
+    private final long gasPrice;
```

Comment:
`fee` and `gas` + `gasPrice` are kind of redundant. `gasPrice` can be derived by `fee / gas`.

Reply:
In this impl, we still have our flat fee of 0.005 sem + gas.  It was an open question 2 month ago, so went with a guess